src/Microsoft.Azure.WebJobs/AutoResolveAttribute.cs

```diff
@@ -25,5 +26,10 @@ public AutoResolveAttribute()
         /// in the Microsoft.Azure.WebJobs.Host assembly.
         /// </summary>
         public Type ResolutionPolicyType { get; set; }
+
+        /// <summary>
+        /// If this is empty, then specify a default. 
+        /// </summary>
+        public DefaultResolveValue Default { get; set; } = DefaultResolveValue.None;
```

Comment:
Initially I was thinking that we wanted this to behave like AppSetting.DefaultValue which allows a string literal to be specified, which will be used if the target doesn't already have a value.

By using this enum, we can't express the same thing here. E.g. what if I want [AutoResolve(DefaultValue = "Mathew")], where "Mathew" is only used if no value comes from binding data, app settings, etc.?